front-end/src/variables.scss

```diff
@@ -19,4 +19,7 @@ $sortArrowActive: #584B82;
 $sortArrowInactive: #62568F;
 $headerTextColor: #444444;
 
+$female: #FF659E;
```

Comment:
We have recently agreed on this approach to color variables naming:
https://github.com/Thealoner/danit-final-project/pull/132#discussion_r240372603